### ISSUE TITLE
Generator loss in Super-resolution GAN.

### ISSUE THREAD

**Issue by tahmid32** [PacktPublishing/Generative-Adversarial-Networks-Projects#102]
Hi there! Thanks for the good work. I am using Super-resolution GAN code from here for my academic project. Here in the code snippet:

            """
            Train the generator network
            """

            # Sample a batch of images
            high_resolution_images, low_resolution_images = sample_images(data_dir=data_dir, batch_size=batch_size,
                                                                          low_resolution_shape=low_resolution_shape,
                                                                          high_resolution_shape=high_resolution_shape)
            # Normalize images
            high_resolution_images = high_resolution_images / 127.5 - 1.
            low_resolution_images = low_resolution_images / 127.5 - 1.

            # Extract feature maps for real high-resolution images
            image_features = vgg.predict(high_resolution_images)

            # Train the generator network
            g_loss = adversarial_model.train_on_batch([low_resolution_images, high_resolution_images],
                                             [real_labels, image_features])

            print("g_loss:", g_loss)

When g_loss is printed, 3 losses are shown. My query is that middle one among the three losses belongs to which category. Thanks and regards.
